hardhat.config.ts

```diff
@@ -1,61 +1,76 @@
-import { HardhatUserConfig } from "hardhat/config";
-import "@nomicfoundation/hardhat-toolbox";
-import "@typechain/hardhat";
-import "@nomicfoundation/hardhat-ethers";
-import "@nomicfoundation/hardhat-chai-matchers";
-import "dotenv/config";
+/** @format */
```

Comment:
Same here. Look at my comment in this PR: https://github.com/Bima-Labs/bima-v1-core/pull/7

If you have trouble matching your prettier to the code, let me know and I will add local prettier settings for the repo itself :)